functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -189,7 +242,14 @@ Then it will export the results to Export-DbaDiagnosticQuery.
             }
 
             if (!$instanceOnly) {
-                $databases = $server.Query("Select Name from sys.databases where name not in ('master', 'model', 'msdb', 'tempdb')")
+                if (-not $DatabaseName) {
+                    $databases = (Get-DbaDatabase -SqlInstance $server -ExcludeAllSystemDb).Name
+                    Write-Message -Level Debug -Message  "No Database Name Filtered: Pulled $(@($Databases).count)" -Verbose
```

Comment:
Why are we including `-Verbose` parameter on this command?

We reserve `-Level Debug` for the purpose of T-SQL statements. You might put this as `-Level VeryVerbose` or `-Level System` just so the logging system picks it up. I don't see any reason to output this type of information as debug.

Reply:
Count property also needs to be capitalized.